neo/rawio/plexon2rawio/plexon2rawio.py

```diff
@@ -53,6 +53,10 @@ class Plexon2RawIO(BaseRawIO):
     pl2_dll_file_path: str | Path | None, default: None
         The path to the necessary dll for loading pl2 files
         If None will find correct dll for architecture and if it does not exist will download it
+    reading_attempts: int, default: 15
+        Number of attempts to read the file before raising an error
+        This opening process is somehow unreliable and might fail occasionally. Adjust this higher
```

Comment:
```suggestion
        This opening process is somewhat unreliable and might fail occasionally. Adjust this higher
```
somewhat gives a stronger flavor of unpredictability in success (so it feels like "sometimes"). somehow implies a more mechanistically nuance. So it would mean more like there is a problem in the implementation that we could figure out and fix, but we haven't yet.

In this case I would tend toward somewhat (and somewhat is more commonly used in general).

Reply:
I was no aware of such nuance. Thanks for that.